src/components/NextPrevious/styles.tsx

```diff
@@ -40,7 +40,7 @@ export const NextPreviousLinkStyle = styled.a`
     width: 0.5rem;
   }
 
-  &:first-child {
+  &:first-of-type {
     img {
       margin-left: 0;
```

Comment:
```suggestion
      margin-left: 1rem;
```

With the change to `:first-of-type`, when there is no Previous, the spacing between text and icon on the Next button is off. Ex. https://new-ssg.dmhjrabi3pkql.amplifyapp.com/lib/analytics/getting-started/q/platform/flutter/#view-analytics-console

<img width="220" alt="Screenshot 2023-10-16 at 1 13 36 PM" src="https://github.com/aws-amplify/docs/assets/30757403/ea501a67-185a-48e0-9aa8-4c1b2f53ee4c">